<!-- -->
<launch>

  <!-- send table.xml to param server -->
  <arg name="use_gnss" default="1" />
  <arg name="queue_size" default="10" />
  <arg name="sync" default="false" />
  
  <node pkg="ndt_localizer" type="ndt_matching" name="ndt_matching" output="screen">
    <param name="use_gnss" value="$(arg use_gnss)" />
    <param name="queue_size" value="$(arg queue_size)"/>
    <remap from="/points_raw" to="/sync_drivers/points_raw" if="$(arg sync)" />
  </node>
  
</launch>
